x-pack/plugins/apm/index.ts

```diff
@@ -35,7 +36,9 @@ export function apm(kibana) {
       },
       styleSheetPaths: resolve(__dirname, 'public/index.scss'),
       home: ['plugins/apm/register_feature'],
-      injectDefaultVars(server) {
+
+      // TODO: get proper types
+      injectDefaultVars(server: any) {
```

Comment:
@epixa Does that platform provide types for this, or the other places where I've used `any` in this file?

Reply:
Folks have been building types for some of these things here: https://github.com/elastic/kibana/blob/master/src/legacy/server/kbn_server.d.ts

Note that none of these things are provided by or available in the new platform, so it's not the thing that provides these types.

Reply:
> Note that none of these things are provided by or available in the new platform, so it's not the thing that provides these types.

This is surprising. Are you saying that type definitions for stuff like `server` and `kibana` are not provided by the New Platform? Aren't these things created by the platform in Typescript in the first place?